cmd/vsphere-xcopy-volume-populator/certificate-tool/assets/manifests/xcopy-setup/controller.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: forklift-volume-populator-controller
+  namespace: ${.TEST_NAMESPACE}
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ${.TEST_LABELS}
+  template:
+    metadata:
+      labels:
+        app: ${.TEST_LABELS}
+    spec:
+      containers:
+        - args:
+            - -v
+            - "5"
+          env:
+            - name: ${.POD_NAMESPACE}
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: metadata.namespace
+            - name: OVIRT_POPULATOR_IMAGE
```

Comment:
this is nice to have, though not a must,  to have the other 2 populators working. @mnecas in case you are interested to take this further.